### PR TITLE
Remove supertypes to document_types conversions

### PR DESCRIPTION
**Pull request by koetsier**
Revert translations of supertypes to content types in search

See: https://github.com/alphagov/finder-frontend/pull/606

Supertypes were translated to content types in the expectation that
supertypes would be deprecated in the search api.

This is actually quite complicated to do (for instance handling cases where
supergroups are 'other') and so we will keep indexing supertypes in
ElasticSearch.

See also: https://github.com/alphagov/rummager/pull/1426

Trello: https://trello.com/c/jInE7P59/362-remove-supertypes-to-documenttypes-conversions

### CODE REVIEW

File: features/support/env.rb [hunk 1]
```diff
@@ -16,6 +16,7 @@
 require 'slimmer/test'
 
 GovukTest.configure
+Capybara.javascript_driver = :selenium_chrome
```
Comment (andysellick) [inline features/support/env.rb]: Was this an attempt to get that failing test to pass? Seems like changing this could have consequences.

Comment (koetsier) [reply features/support/env.rb]: yes, for debugging - this is a WIP

File: features/fixtures/statistics.json [hunk 2]
```diff
@@ -25,7 +25,6 @@
   "details":{
     "document_noun":"statistic",
     "filter":{
-      "content_purpose_supergroup": "research_and_statistics"
```
Comment (bilbof) [inline features/fixtures/statistics.json]: Does the stats supergroup finder no longer need this field?

File: features/support/rummager_url_helper.rb [hunk 3]
```diff
@@ -48,57 +48,17 @@ def policy_search_fields
...
-      utaac_decision
-      world_location_news_article
-      world_news_story
-      written_statement
-    )
-
     base_search_params.merge(
       'fields' => news_and_communications_search_fields.join(','),
-      'filter_content_store_document_type' => supergroup_document_types,
+      'filter_content_purpose_supergroup' => 'news_and_communications',
```
Comment (bilbof) [inline features/support/rummager_url_helper.rb]: Really good change.

File: features/fixtures/statistics.json (hunk 2 above)
Comment (koetsier) [reply features/fixtures/statistics.json]: The test doesn't really - this was part of testing cps->content types conversion

**Pull request merged**